bmu/Src/batteries.c

```diff
@@ -101,7 +101,7 @@ float voltageToSOCLookup[NUM_SOC_LOOKUP_VALS] = {
 /*
  * HV Measure task Defines and Variables
  */
-#define HV_MEASURE_TASK_PERIOD_MS 1
+#define HV_MEASURE_TASK_PERIOD_MS 5
```

Comment:
Don't feel good about this change, is this rules compliant? I forget the freq we need

Reply:
I might be wrong, but I checked Hybrid (2023) and fsae (2024 V1) rules and I couldn't find anything regarding sampling frequency